[2024-11-18T19:03:28.637+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T19:03:29.381+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.accurate scheduled__2024-11-14T00:00:00+00:00 [queued]>
[2024-11-18T19:03:29.962+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.accurate scheduled__2024-11-14T00:00:00+00:00 [queued]>
[2024-11-18T19:03:30.009+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-18T19:03:31.025+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): accurate> on 2024-11-14 00:00:00+00:00
[2024-11-18T19:03:32.087+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T19:03:32.136+0000] {standard_task_runner.py:72} INFO - Started process 122 to run task
[2024-11-18T19:03:32.149+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'accurate', 'scheduled__2024-11-14T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/hello_world_dag.py', '--cfg-path', '/tmp/tmplje01jo_']
[2024-11-18T19:03:32.428+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask accurate
[2024-11-18T19:04:21.477+0000] {task_command.py:467} INFO - Running <TaskInstance: hello_world_dag.accurate scheduled__2024-11-14T00:00:00+00:00 [running]> on host e14bbb8637e8
[2024-11-18T19:04:21.654+0000] {job.py:229} INFO - Heartbeat recovered after 55.16 seconds
[2024-11-18T19:04:23.590+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='accurate' AIRFLOW_CTX_EXECUTION_DATE='2024-11-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-14T00:00:00+00:00'
[2024-11-18T19:04:23.604+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T19:04:23.604+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T19:04:23.606+0000] {logging_mixin.py:190} INFO - Current task name:accurate state:running start_date:2024-11-18 19:03:29.441033+00:00
[2024-11-18T19:04:23.606+0000] {logging_mixin.py:190} INFO - Dag name:hello_world_dag and current dag run status:running
[2024-11-18T19:04:23.607+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T19:04:23.648+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-18T19:04:23.649+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'accurate'"]
[2024-11-18T19:04:23.849+0000] {subprocess.py:99} INFO - Output:
[2024-11-18T19:04:23.909+0000] {subprocess.py:106} INFO - accurate
[2024-11-18T19:04:23.915+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-18T19:04:24.062+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T19:04:24.086+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=accurate, run_id=scheduled__2024-11-14T00:00:00+00:00, execution_date=20241114T000000, start_date=20241118T190329, end_date=20241118T190424
[2024-11-18T19:04:24.189+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T19:04:24.191+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T19:04:24.192+0000] {logging_mixin.py:190} INFO - Dag name:hello_world_dag queued_at:2024-11-18 19:02:05.015845+00:00
[2024-11-18T19:04:24.194+0000] {logging_mixin.py:190} INFO - Task hostname:e14bbb8637e8 operator:BashOperator
[2024-11-18T19:04:24.376+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T19:04:24.419+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
